tooling/build_changelog

```diff
@@ -0,0 +1,10 @@
+the_dir=$1
+: ${the_dir:=.}
+cd $the_dir
+pwd
+# We run the script twice.  First time it to create the base file:
+curl -s "https://raw.githubusercontent.com/release-depot/change/latest/change" |
+sh -s -- init
+# Second time is to populate it with tag data:
```

Comment:
Perhaps one temporary solution is to comment this out until we have added some tag in a future commit?